calculator.php

```diff
@@ -149,12 +133,14 @@ function show_info_block()
     info('History commands:' . PHP_EOL . 'Full - to see full history.' . PHP_EOL . 'Format of date "01-01-1990" - to show history of current date.');
 }
 
-function show_history($history)
+function show_history($historyRepository)
 {
-    if ($history === null) {
-        info('You have no history');
+    $history = $historyRepository->fileGetContent();
```

Comment:
please implement new method `all` inside the HistoryRepository class, which will return array with all history items from the `history.json` file

```suggestion
    $history = $historyRepository->all();
```